test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -660,6 +660,7 @@ def _cleanup(self):
             ftd.extend([f for f in files if bn(f).startswith('pbsscrpt')])
             ftd.extend([f for f in files if bn(f).startswith('pbs.conf.')])
             ftd.extend([f for f in files if p.match(bn(f))])
+            ftd.extend([f for f in du.tmpfilelist])
```

Comment:
You will also have to clear the `du.tmpfilelist` when you're done cleaning it; otherwise the next time `_cleanup` is called it will try to delete the same files again. I realize that it only gets called in `begin` and `finalize`, but it's a simple change to prevent any problems in the future.